### DISCUSSION TITLE
Adding Jitsi Servers - Docker

### DISCUSSION THREAD

**Discussion by ghost** [H2-invent/jitsi-admin#544] [Q&A]
Hey everybody,

we set up the .env file so only people in the group keycloak group "admin" can add jitsi servers to jitsi-admin.

laf_CreateServerOpenToEveryone=0
laf_keycloakGroupToCreateServer=/admin

I created a user and he cannot add any server. Problem now is, that even if I create the keycloak group "admin" and add the user to this group, I cannot add a server.

I assume that the change of the group is not recognized by the jitsi-admin but I don't know how I can fix that.

We want to use Jitsi-Admin for our small business but want to prevent users from adding any servers themselves.

Thanks for any advice.

Edit: next problem: when we restart the server, the jitsi-admin-app-ja-1 container won't start again "Acces denied for user jitsiadmin". So it is not possible to ever restart without destroying the whole tool.

**Comment by holzi1005** [comment]
Hello @tP1812 
you should add an mapper in the Keycloak Client so Jitsi Admin will get the information about the users:

https://github.com/H2-invent/jitsi-admin/wiki/Organize-Jitsi-Servers-via-keycloak-groups
